api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -241,38 +241,48 @@
 
     /**
      * <p>
-     * Normally if an LRA is present when a bean method is executed it will not
-     * be ended when the method returns. To override this behaviour and force LRA
-     * termination on exit use the end element.
+     * Normally if an LRA is present when a resource method is executed it will not
+     * be ended when the method returns. To override this behaviour and force the LRA
+     * to close on exit use this <code>end</code> element with the value set to
+     * {@literal true}.
      * </p>
      *
      * <p>
-     * If the <code>end</code> value is set to <code>false</code>
-     * while the annotated method returns the {@link Response} status code
-     * with the meaning the LRA to be cancelled (see {@link #cancelOn()} and {@link #cancelOnFamily()})
-     * then this error state has precedence over the <code>end</code> attribute.
-     * It means the LRA will be cancelled despite the end was set to <code>false</code>.
+     * If the <code>end</code> value is set to {@literal false}
+     * but the annotated method finishes with a status code that matches
+     * one of the {@link #cancelOn()} and {@link #cancelOnFamily()} values then
+     * the LRA will be cancelled instead of closed. In other words the
```

Comment:
```suggestion
     * the LRA will be cancelled. In other words the
```
this gives wrong perception that even when end = false then the LRA is by default expected to be closed.

Reply:
Not sure what your point is but I'd prefer to use my text.

Reply:
reading from the beginning of the paragraph, we have `@LRA(end = false)` so it should not end when the method is finished. If status code matching cancel* attributes is returned then LRA will be cancelled. But that doesn't mean it will be cancelled instead of closed because it should not be closed when the method is finished because of the end=false.

But I understand that it should be by default ended in some other method later. This is just not clear from the sentence and it may be confusing for someone reading the spec top bottom. Keep it as you wish.

Reply:
You are correct, +1